scripts/carthage.sh

```diff
@@ -0,0 +1,21 @@
+# This script is needed to use Carthage in Xcode 12, since it currently has a compatibility issue
```

Comment:
turns out we still need this one, unfortunately.

Reply:
😢